### PR TITLE
Adding Simple Features for SQL Mappings to Annex C

### PR DESCRIPTION
**Pull request by jabhay**
We decided that we wanted to add mappings to other OGC Standards. This Pull Request adds those mappings for Simple Features for SQL.

### CODE REVIEW

**Comment by situx** [comment]
Could we create links for the URIs in the table since they should be dereferenceable in the future?

**Review by situx — APPROVED** [review]
I added some links and changed the table layout. Could you also add a reference to the Simple features specification document?
Then I believe it would be good to go.

File: 1.1/spec/18-Annex-C.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+The following table maps the functions and properties from Simple Features for SQL to GeoSPARQL.
+
+[frame=none, grid=none, cols="3,3,3,3,3"]
+|===
+| Simple Features for SQL | GeoSPARQL Equivalent | Since GeoSPARQL | Related Property Available | Since GeoSPARQL
+| 2.1.1.1 Basic Methods on Geometry | | | |
+| Dimension(): Double | N/A | - | http://www.opengis.net/ont/geosparql#dimension[geo:dimension] | 1.0
+| GeometryType(): Integer | Class of geometry instance | 1.0 | N/A | -
+| SRID(): Integer | http://www.opengis.net/def/function/geosparql/getSRID[geof:getSRID] | 1.0 | http://www.opengis.net/ont/geosparql#inSRS[geo:inSRS] | 1.1
```
Comment (mperry455) [inline 1.1/spec/18-Annex-C.adoc:14]: I guess we need to remove inSRS if the group decides to move this property to GeoSPARQL 1.2

**Comment by mperry455** [comment]
There are several simple features functions that could be added as SPARQL functions and/or RDF properties (isEmpty, isSimple, boundary, envelope, centriod, length, area). Right now, some are only functions, some are only properties, and some are both. I wonder if we should do something uniform here and have both a function and a property for each one. The choice of function or property looks kind of arbitrary at the moment.

**Review by mperry455 — APPROVED** [review]

**Comment by situx** [comment]
I have removed the inSRS property from the table because we decided in the last meeting not to include it. As the table should show what is implemented until GeoSPARQL 1.1 I would not want to keep speculative references to GeoSPARQL 1.2 in the GeoSPARQL 1.1 standard.
Concerning aligning properties and functions it would make sense to define at least pairs of functions and properties where applicable.

**Comment by situx** [comment]
I added the missing reference and will merge this request for now as we have two approvals. Possibly adding further functions is a separate pull request anyway.

**Pull request merged**